### ISSUE TITLE
Anyone knows why from this website cannot be created movies? sdo.gsfc.nasa.gov/data/aiahmi/

### ISSUE THREAD

**Issue by Anthrwpos** [Helioviewer-Project/api#231]
I know here isn't for this website but i don't know where else to write about it. For over a week, this site cannot create movies from data. Is down or i'm the only who have this problem? sdo.gsfc.nasa.gov/data/aiahmi/

**Comment by dgarciabriseno** [comment]
The maintainers aren't online right now so I emailed them. I'll try to remember to follow up.

**Comment by dgarciabriseno** [comment]
It looks like it's working now. Can you confirm it's working for you @Anthrwpos ?

**Comment by Anthrwpos** [comment]
Yes, as far as I can see it is working now.  When you say you e-mailed them, is that something between you and them, or can anyone e-mail the group?
Thanks

**Comment by dgarciabriseno** [comment]
You can email them, their names and emails are at the bottom of the page
<img width="439" alt="image" src="https://user-images.githubusercontent.com/94071409/209144130-7797d05f-a122-4434-9c62-f829773c9879.png">

It looks like the email address is slightly obfuscated to avoid spam emails. You have to hover over the name and it will give you the email address. I don't think Dean's is actually his email though. But Kevin's is correct.

**Comment by Anthrwpos** [comment]
> You can email them, their names and emails are at the bottom of the page <img alt="image" width="439" src="https://user-images.githubusercontent.com/94071409/209144130-7797d05f-a122-4434-9c62-f829773c9879.png">
> 
> It looks like the email address is slightly obfuscated to avoid spam emails. You have to hover over the name and it will give you the email address. I don't think Dean's is actually his email though. But Kevin's is correct.

Thank you for contacting them. It works now.

> The maintainers aren't online right now
Regarding this, isn't the data generated automatically? Νeed a human presence to release the data every time, so someone can create a video?

**Comment by Anthrwpos** [comment]
Hi again and sorry. But i contact a couple of days ago with Dean Pesnell, but an error seem to happening again: _[APPLICATION ERROR] Caught exception: SQLSTATE[HY000] [1130] Host 'gs671-sdoweb.ndc.nasa.gov' is not allowed to connect to this MySQL server_ His email is [this](https://science.gsfc.nasa.gov/sed/bio/william.d.pesnell)?
